app/services/solr_facade_service.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
```

Comment:
I'm not sure this service object adds that much to the embedded SolrFacade class.  Could be just have the SolrFacade object without this wrapper?